.eslintrc.js

```diff
@@ -21,9 +23,10 @@ module.exports = {
     ecmaVersion: 2018,
     sourceType: 'module',
   },
-  plugins: ['react', 'react-hooks'],
+  plugins: ['react', 'react-hooks'.'prettier'],
```

Comment:
I don't understand this syntax. Should this be a comma?

Reply:
Oops its a mistake, should be a comma!